pages/api/v1/discord/event.ts

```diff
@@ -46,8 +55,18 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse<
       ":bangbang: New Discord Event Created :bangbang:",
       `${process.env.ACM_DISCORD_INVITE_URL}?event=${eventID}`
     );
+    
+    // Add cover image, if included
+    var cover = ""
+    if(image){
+      await discord.addCoverEvent(
+        eventID,
+        image,
+      );
+    }
+    else cover = ", without a cover image";
```

Comment:
For variables where we need to modify values, use `let` instead of `var`, `var` can lead to weird and zany bugs

I also don't think we need to update a cover variable here, instead below we can do 
```ts
return res.status(200).json({ 
  message: image ? `Event created successfully!` : `Event created successfully without a cover image!` 
});
```
I also don't think we need a special indication in the message if we uploaded an image or not tbh